aspnetcore/fundamentals/minimal-apis/7.0-samples/todo-group/Program.cs

```diff
@@ -0,0 +1,201 @@
+using Data;
+using Microsoft.Data.Sqlite;
+using Microsoft.EntityFrameworkCore;
+
+var builder = WebApplication.CreateBuilder(args);
+
+// Learn more about configuring Swagger/OpenAPI at https://aka.ms/aspnetcore/swashbuckle
+builder.Services.AddEndpointsApiExplorer();
+builder.Services.AddSwaggerGen();
+var connection = new SqliteConnection("DataSource=:memory:");
+connection.Open();
+// Add InMemoryDatabase to the container.
+builder.Services.AddDbContext<ApplicationDbContext>(options =>
```

Comment:
Nit: I prefer keeping the lines that add services all together.

```suggestion
var builder = WebApplication.CreateBuilder(args);
// Create and open an in-memory SQLite connection.
var connection = new SqliteConnection("DataSource=:memory:");
connection.Open();

// Learn more about configuring Swagger/OpenAPI at https://aka.ms/aspnetcore/swashbuckle
builder.Services.AddEndpointsApiExplorer();
builder.Services.AddSwaggerGen();
// Configure ApplicationDbContext to use the in-memory SQLite connection and register it as a service.
builder.Services.AddDbContext<ApplicationDbContext>(options =>
```